CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
The `Salary` property should be nullable (`decimal?`) if it's optional. Otherwise, ensure all creation/update flows provide a value to avoid runtime errors or validation failures.

```suggestion
public decimal? Salary { get; set; }
```